workflow-core/src/main/java/com/squareup/workflow/WorkflowAction.kt

```diff
@@ -73,12 +76,12 @@ interface WorkflowAction<StateT, out OutputT> {
      * Use this to, for example, ignore the output of a child workflow or worker.
      */
     @Suppress("UNCHECKED_CAST")
-    fun <StateT, OutputT> noAction(): WorkflowAction<StateT, OutputT> =
-      NO_ACTION as WorkflowAction<StateT, OutputT>
+    fun <PropsT, StateT, OutputT> noAction(): WorkflowAction<PropsT, StateT, OutputT> =
+      NO_ACTION as WorkflowAction<Any?, StateT, OutputT>
 
     /**
-     * Convenience function that returns a [WorkflowAction] that will just set the state to [newState]
-     * (without considering the current state) and optionally emit an output.
+     * Convenience function that returns a [WorkflowAction] that will just set the state to
+     * [newState] (without considering the current state) and optionally emit an output.
      */
     @Deprecated(
```

Comment:
@zach-klippenstein  We should delete these, they're so dangerous. Is that tracked?

Reply:
I don't think so. We don't have any issues for removing deprecated APIs afaik.

Reply:
These are the only ones I really care about. After this merges I'll put up a PR to snip them. Not worth tracking.

Reply:
I thought they were still extremely popular.

Reply:
They are, but they're also often extremely a bad idea, and misused more often than not. And easy enough to recreate on your own w/o us sanctioning them.

Reply:
https://github.com/square/workflow-kotlin/pull/85